h/static/scripts/widget-controller.js

```diff
@@ -117,6 +142,25 @@ module.exports = function WidgetController(
     crossframe.call('scrollToAnnotation', annotation.$$tag);
   }
 
+  /** Returns the annotation type - note or annotation of the first annotation
+   *  in `results` whose ID is a key in `selectedAnnotationMap`.
+   */
+  function tabTypeFromSelection(selection, results) {
```

Comment:
I'm trying to follow a convention in new code that either: 1) All paths through the code return no value, in which case it always uses just `return;` or 2) All paths through the code explicitly return a value, in which case either `return <value>` or `return null;`. This makes it easier to spot a mistake where some path fails to return a value but should do.